bundler.d/test.rb

```diff
@@ -13,5 +13,7 @@
   gem 'factory_girl_rails', '~> 4.5', :require => false
   gem 'rubocop-checkstyle_formatter', '~> 0.1'
   gem "poltergeist"
-  gem 'test-unit', :platforms => :ruby_22
+  if RUBY_VERSION >= '2.2'
```

Comment:
i thought this simply requires gem update bundler ?

but i assume that will work on older bundlers so :+1: